app/controllers/venues_controller.rb

```diff
@@ -0,0 +1,7 @@
+class VenuesController < ApplicationController
+
+  def show
+    @venue = Venue.find_by(id: params[:id])
+    @events = Event.active_events(@venue)
```

Comment:
Would this would work as well to allow to remove the class method and utilize the scopes we have in play already?

@events = @venue.events.open_events